### PR TITLE
feature: add transform-and-upload.py to publish assets

### PR DESCRIPTION
**Pull request by ds-lcapellino**
## Description
- add python script to publish assets
- add corresponding documentation how to use the script

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: scripts/transform-and-upload.py
Comment (SebastianBezold) [inline scripts/transform-and-upload.py:1]: Copyright and License header missing

**Review by SebastianBezold — COMMENTED** [review]
Hi @ds-lcapellino,

nice that there is some progress on this topic 👍 
Not really familiar with python, so I cannot give any feedback on that.
With my knowledge, it seems like only built in libraries are used, so no 3rd party IP checks needed, right?

**Comment by SebastianBezold** [comment]
Hey @ds-lcapellino,

to document for everyone else: You mentioned, that only the [requests lib](https://pypi.org/project/requests/) is not part of the Python core. I found an [IPlab issue](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/13074), that is already approved, for the exact version `2.31.0` you are using. So this is fine for me. 
Since we do not have an automated way to create a `DEPENDENCIES` file via dash, I'll add the repsective info to the `NOTICE.md` after merging this PR.

Could you please double check, if I missed something?

**Comment by ds-lcapellino** [comment]
Hey @SebastianBezold,
I rechecked and there are only standard modules from Python used besides requests:2.31.0. 
I will wait for an Eclipse Committer with good Python skills to review the Python file.

**Comment by SebastianBezold** [comment]
Since no one could do a review on the python script yet, I'll take it as is and add the dependency annotations to `NOTICE.md` right after merging this PR

**Pull request merged**